.buildkite/scripts/common/setup_node.sh

```diff
@@ -32,7 +32,16 @@ elif [[ "$UNAME" == "Darwin" ]]; then
 fi
 echo " -- Running on OS: $OS"
 
-nodeUrl="https://us-central1-elastic-kibana-184716.cloudfunctions.net/kibana-ci-proxy-cache/dist/v$NODE_VERSION/node-v$NODE_VERSION-${OS}-${classifier}"
+NODE_VARIANT=""
```

Comment:
Note: until we remove Bazel we will need to update this as part of our node upgrade Work